architecture/2-UserStories/2-MatchingProtocols.yaml

```diff
@@ -0,0 +1,10 @@
+id: 2
+name: Matching Protocols
+actor: Clinician
+behavior: {activity-that-manages-matching-protocols}
```

Comment:
@mmroden what should we put here as far as what the tetra does to maintain matching protocols per patient volume?

Reply:
Right now, that doing so is part of the testing protocol (ie, that the entire configuration will stay at the PIP/PEEP it's set at when left for several hours).

Reply:
is it possible to encapsulate the testing protocol into a sentence here?